woocommerce/woocommerce-android
# Feature/9811 open tone bottom sheet

Closes: #9811


### Description

Add select tone bottom sheet and save selected value in main viewmodel

### Testing instructions

1. Click on "Add Product" button from products list and select add product with AI. The site must be WP.com to be eligible for this option. 
2. Click "Continue" in the first step. 
3. Click on "Set tone and voice" 
4. Check the UI matches the design
5. Select any tone
6. Click again on "Set tone and voice" and check your previous selected option is now selected. 

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/2663464/32357e10-5323-4c97-bbc9-d9afa459c7fe
